packaging/sunshine.rb

```diff
@@ -1,6 +1,7 @@
 require "language/node"
 
 class @PROJECT_NAME@ < Formula
+  # conflicts_with "sunshine-beta", because: "sunshine and sunshine-beta cannot be installed at the same time"
```

Comment:
I think the right way to handle it is, for Stable sunshine, it should be `sunshine-beta`. For Beta sunshine, it should be `sunshine`. So if you're commenting out for Stable, it should be just `sunshine` here.

Reply:
Commenting out cause it fails the audit. It's not commented out in the beta formula that gets published though.

Anything more advanced than this will require changes to https://github.com/LizardByte/homebrew-release-action

But yea, the conflict needs to be reversed with this logic.